troposphere/static/js/utilities/clipboardFunctions.js

```diff
@@ -76,34 +50,30 @@ const copyElement = (element, options) => {
     let copied = false;
     if (hasClipboardAPI()) {
         let range = document.createRange(),
-            selection = document.getSelection();
+        selection = document.getSelection();
```

Comment:
I think the ident here is correct since it is using `let` to define two variables? It would be correct if there is no `,` after `createRange()` I believe.